app/controllers/application_controller.rb

```diff
@@ -0,0 +1,24 @@
+class ApplicationController < ActionController::Base
+    helper_method :current_user, :human_readable_size, :private_route
+
+    def current_user 
+        @current_user ||= User.find(session[:user]) if session[:user]
+    end
+    
+    def private_route 
+        redirect_to '/sign_in' unless current_user 
+    end 
+
+    def human_readable_size(size)
```

Comment:
Ideally such a method which is used for UI rendering should be in `<controller>_helper` file

Reply:
Moved this to document_controller file